ios/Constants/CardVerificationChoiceConstants.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CardVerificationChoiceConstants.swift
+//  fidel-react-native
+//
+//  Created by Valeria Rogatchevskikh on 12/6/23.
+//
+
+import Foundation
+import Fidel
+
+extension CardVerificationChoice: ConstantsProvider {
+    
+    var constantKey: String {
+        switch self {
+        case .onTheSpot: return "onTheSpot"
+        case .delegatedToThirdParty: return "delegatedToThirdParty"
+        @unknown default: return "unexpected"
+        }
+    }
+    
+    public static var allCases: Set<CardVerificationChoice> = [CardVerificationChoice.onTheSpot, .delegatedToThirdParty]
+    static var parentKeyName: String = String(describing: Self.self)
+    
+    static func verificationChoice(from constantKey: String?) -> CardVerificationChoice? {
```

Comment:
Is this function used anywhere?

Reply:
you were right, not used, so I removed it